app/views/users/_attr.html.erb

```diff
@@ -6,7 +6,9 @@
         <% elsif attr == "mail" %>
             <div class="attr"><a href="mailto:<%= value[0] %>"><%= value[0] %></a></div>
         <% elsif attr == "mobile" %>
-            <div class="attr"><a href="tel:<%= value[0].tr("() -", "") %>"><%= value[0] %></a></div>
+	    <div class="attr"><a href="tel:<%= value[0].tr("() -", "") %>"><%= value[0] %></a></div>
+	<% elsif attr == "github" %>
+            <div class="attr"><a href="https://github.com/:<%= value[0].tr("() -", "") %>"><%= value[0] %></a></div>
```

Comment:
I do not think that there should be a `:` inside the URL. There `:` is a HTML thing and not a ruby thing.